releasetasks/templates/desktop/binary_transparency.yml.tmpl

```diff
@@ -0,0 +1,38 @@
+{% set buildername = "release-{}-{}_binary_transparency".format(branch, product) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    requires:
+        - "{{ stableSlugId('release-foo-{}_chcksms'.format(product)) }}"
+    reruns: 5
+    task:
+        provisionerId: "scriptworker-prov-v1"
+        workerType: "dummy-worker-transpar"
```

Comment:
Are you gonna switch these to real workerTypes in a follow-up commit are you relying on these dummy workers for now and address these later as you rollback binarytransparency fully productional?